rust/arrow/src/ipc/file/reader.rs

```diff
@@ -530,7 +530,7 @@ mod tests {
         let testdata = env::var("ARROW_TEST_DATA").expect("ARROW_TEST_DATA not defined");
         // the test is repetitive, thus we can read all supported files at once
         let paths = vec![
-            // "generated_datetime",
+            "generated_datetime",
```

Comment:
with this change, we can now read the datetime integration files, which were failing because of timezones.